inst/workflow/4_modules/report_kri_site.yaml

```
@@ -1,5 +1,6 @@
 meta:
-  File: report_kri_site.yaml
+  Type: Report 
+  ID: report_kri_site
```

```suggestion
  ID: report_kri_site
```
```suggestion
  ID: report_kri_site
  Output: html
```
To match above `report_kri_country.yaml`